qml/pages/PackagePage.qml

```diff
@@ -81,13 +81,11 @@ Page {
           }
       }
 
-      ChumDetailItem {
```

Comment:
:) I would have put it back. See also official Store - it has that "Sailfish app" on top.